using-kubecost/navigating-the-kubecost-ui/teams.md

```diff
@@ -23,79 +22,236 @@ persistentVolume:
 
 ```yaml
-saml:
+oidc:
   enabled: true
   rbac:
-    enabled: false
+    enabled: true
 ```
 
-The members of your team are now able to add new members, or create new teams.
+The first person to log in will be added to a default team with admin-level access. They will then be able to configure teams through the UI on the Teams page.
 
-### Method 2: Disable SAML RBAC while leaving SAML enabled
+### Enable Teams and Define in Helm
 
-This method leaves additional authentication in place while SAML RBAC is disabled.
+{% hint style="warning" %}
+Configuring teams through Helm will disable configuration of teams through the UI.
+{% endhint %}
+
+
+Kubecost allows teams to be defined in Helm values obviating the need to create them in the UI. Note that Teams must still be enabled using Helm regardless of if configuration is done in the UI or via Helm by enabling RBAC within your desired protocol config, as [above](teams.md#enable-teams-and-define-in-ui).
+
+An example values snippet is shown below for the creation of a team in Helm.
 
 ```yaml
-saml:
-  enabled: true
-  rbac:
-    enabled: false
+ teams:
+   # teamsConfigMapName: teams-configmap-name
+   teamsConfig:
+     - id: ''
+       name: helm-team
+       roles:
+       - id: ''
+         name: helm-role
+         description: helm configured role
+         pages:
+           showOverview: true
+           showAllocation: true
+           showAsset: true
+           showCloudCost: true
+           showClusters: true
+           showExternalCosts: true
+           showNetwork: true
+           showCollections: true
+           showReports: true
+           showInsights: true
+           showActions: true
+           showAlerts: true
+           showBudgets: true
+           showAnomalies: true
+           showEfficiency: true
+           showSettings: true
+         permissions: admin
+         routes: []
+         allocationFilters:
+         - key: cluster
+           operator: ":"
+           value: cluster-one
+         assetFilters: []
+         cloudCostFilters: [] 
+       claims:
+         NameID: email@domain.com
 ```
 
-Now, access the Teams page, and create your initial admin team. See the [Adding a team](teams.md#adding-a-team) section below for specific instructions for using the UI. Once your admin team has been created, reenable SAML RBAC authentication:
+
+
+Configure at least one team under `teamsConfig` with an associated role under the team's `roles`.
+
+Alternatively, a ConfigMap can be created manually, with `teamsConfigMapName` set to this ConfigMap's name.
 
 ```yaml
-saml:
-  enabled: true
-  rbac:
-    enabled: true
+apiVersion: v1
+data:
+  rbac-teams-configs.json: '[json teams data]'

```

Comment:
Can you please provide an example of this JSON data? Or a simple instruction on how users would generate it for themselves? Ex., could they just populate the relevant (YAML) section of the Helm values under `teams.teamsConfig` and then convert it to JSON with `yq` (`yq '.teams.teamsConfig' -o json values.yaml`) ?

Reply:
Added an example.